dev-preview/README.md

```diff
@@ -199,3 +200,172 @@ The OpenShift Cluster API Operator deploys the main components (CAPI & CAPA depl
+```
+Congratulations! You have set up the environment correctly.
+
+### Integrate Management Fabric
+
+When you follow the [document](https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.12/html/multicluster_global_hub/multicluster-global-hub#global-hub-importing-managed-hub-in-default-mode) to import a managed hub cluster into the global hub, you should be able to see the cluster cloudevents in Kafka `kessel-inventory` topic.
+When you create a policy in the managed hub cluster, you can see the events including policy and relationships. The results look like this:
+```
+{
+    "specversion": "1.0",
+    "id": "e79857fa-8d00-11ef-ab88-0a580a8002c8",
+    "source": "http://localhost:8081",
+    "type": "redhat.inventory.resources.k8s-policy.created",
+    "subject": "/resources/k8s-policy/open-cluster-management-global-set/test-local",
+    "datacontenttype": "application/json",
+    "time": "2024-10-18T03:27:29.676142591Z",
+    "data": {
+        "metadata": {
+            "id": 1,
+            "last_reported": "2024-10-18T03:27:29.676142591Z",
+            "resource_type": "k8s-policy",
+            "workspace": "",
+            "labels": null
+        },
+        "reporter_data": {
+            "reporter_instance_id": "mgdhub-1-client",
+            "reporter_type": "ACM",
+            "last_reported": "2024-10-18T03:27:29.677911526Z",
+            "local_resource_id": "open-cluster-management-global-set/test-local",
+            "reporter_version": "2.11.3",
+            "console_href": "",
+            "api_href": ""
+        },
+        "resource_data": {
+            "disabled": false,
+            "severity": "MEDIUM"
+        }
+    }
+}
+
+{
+    "specversion": "1.0",
+    "id": "ea695dc3-8d00-11ef-ab88-0a580a8002c8",
+    "source": "http://localhost:8081",
+    "type": "redhat.inventory.resources_relationship.k8s-policy_ispropagatedto_k8s-cluster.updated",
+    "subject": "/resources_relationship/k8s-policy_ispropagatedto_k8s-cluster/open-cluster-management-global-set/test-local",
+    "datacontenttype": "application/json",
+    "time": "2024-10-18T03:27:34.406482298Z",
+    "data": {
+        "metadata": {
+            "id": 0,
+            "last_reported": "2024-10-18T03:27:34.406482298Z",
+            "relationship_type": "k8s-policy_ispropagatedto_k8s-cluster"
+        },
+        "reporter_data": {
+            "reporter_instance_id": "mgdhub-1-client",
+            "reporter_type": "ACM",
+            "last_reported": "2024-10-18T03:27:34.406482298Z",
+            "subject_local_resource_id": "open-cluster-management-global-set/test-local",
+            "object_local_resource_id": "managedcluster-1",
+            "reporter_version": "2.11.3"
+        },
+        "relationship_data": {
+            "status": "VIOLATIONS",
+            "k8s_policy_id": 0,
+            "k8s_cluster_id": 0
+        }
+    }
+}
+```
```

Comment:
Why not also add the same message :)
Congratulations! You have set up the environment correctly.